### PR TITLE
#1069 Add isStrictlyBlank and isNotStrictlyBlank

### LINKED ISSUE #1069: null is not blank?!
#### Summary
As far as I know, null is considered as "Blank" (as ["Blank"](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isBlank-java.lang.CharSequence-) is superset of null, empty string or blank characters only)

#### Example
This passes:
```
        String s = null;
        assertThat(s).isNotBlank();
```

#### Java 8 specific ?
* NO  : create Pull Request from the `2.x` branch

### PR DESCRIPTION
**Pull request by ChrisCanCompute**
#### Check List:
* Fixes #1069
* Unit tests : Yes
* Javadoc with a code example (API only) : Yes

Adds isStrictlyBlank and isNotStrictlyBlank methods (which work the same as isBlank and isNotBlank, but with null counting as blank).

### CODE REVIEW

File: src/main/java/org/assertj/core/internal/Strings.java [hunk 1]
```diff
@@ -66,6 +66,7 @@
 import java.util.regex.Pattern;
 import java.util.regex.PatternSyntaxException;
 
+import com.sun.istack.internal.NotNull;
```
Comment (PascalSchumacher) [inline src/main/java/org/assertj/core/internal/Strings.java]: This stray import is causing a compile error on traivs. :(

Comment (ChrisCanCompute) [reply src/main/java/org/assertj/core/internal/Strings.java]: I've been doing Android too long, I'll remove it.

Comment (PascalSchumacher) [reply src/main/java/org/assertj/core/internal/Strings.java]: Thanks!

**Comment by ChrisCanCompute** [comment]
Closing this, as the requirements have changed slightly.

**Pull request closed without merging**